bin/assigner/src/main.cpp

```diff
@@ -128,13 +128,26 @@ void print_circuit(const circuit_proxy<ArithmetizationType> &circuit_proxy,
         used_lookup_tables.push_back(lookup_tables[it]);
     }
 
+
+    // fill public input sizes
+    nil::crypto3::marshalling::types::public_input_sizes_type<TTypeBase> public_input_sizes;
+    using public_input_size_type = typename nil::crypto3::marshalling::types::public_input_sizes_type<TTypeBase>::element_type;
+    std::uint32_t public_input_size = table_proxy.public_inputs_amount();
+    for (std::uint32_t i = 0; i < public_input_size; i++) {
```

Comment:
Unusual type selection for a loop :)

Reply:
Just align with `public_inputs_amount()` result value (https://github.com/NilFoundation/zkllvm-blueprint/blob/fde02cb041a0d4603dd80b9318a66179d98acd65/include/nil/blueprint/blueprint/plonk/assignment_proxy.hpp#L223)